modules/ROOT/pages/scalability/sharded-property-databases/data-ingestion.adoc

```diff
@@ -50,6 +52,35 @@ OPTIONS {
 };
 ----
 
+[role=label--new-2025.12]
+==== Import using local file system
+You can import data into a Neo4j cluster that has no access to any cloud.
+
+. Using the `neo4j-admin database import` command, import data into the `foo-sharded` database, creating one graph shard and three property shards.
+If the `neo4j-admin` process is running on the same server as a Neo4j DBMS process, the Neo4j DBMS process must be stopped.
++
+[source, shell]
+----
+neo4j-admin database import full foo-sharded --nodes=nodes.csv --nodes=movies.csv --relationships=relationships.csv --input-type=csv --property-shard-count=3 --schema=schema.cypher
+----
+
+. Using allow and deny database allocate a single shard to each server in the cluster.
+See xref:clustering/databases.adoc#cluster-allow-deny-db[Controlling locations with allowed/denied databases]
+. Move the produced backups from the local file system of the machine used for the import to the servers hosting each of the shards.
+Each server should have one backup, and the backups must reside in the same path on each server.
+. On each server, update the _neo4j.conf_ to include the correct settings for file seeding as outlined in xref:database-administration/standard-databases/seed-from-uri.adoc[Create a database from a URI].
+. Create the database `foo-sharded` as a sharded property database by seeding it from your backups in the servers file systems:
++
+[source, cypher]
+----
+CREATE DATABASE `foo-sharded`
+DEFAULT LANGUAGE CYPHER 25
+PROPERTY SHARDS { COUNT 3 }
+OPTIONS {
+ seedUri: `file:/backusp/`, seedOptions: 'NO_CHECK'
```

Comment:
This is the first time we mention `NO_CHECK`. Maybe we should add a sentence or two on what this option does.